src/PuliPlugin.php

```diff
@@ -466,10 +466,10 @@ private function loadPuliPackages()
         $packages = array();
 
         $output = $this->puliRunner->run(
-            'package --list --format \'%name%;%installer%;%install_path%;%state%\''
+            'package --list --format "%name%;%installer%;%install_path%;%state%"'
```

Comment:
So what you're saying is that single quotes for escaping never work on Windows? This is used in other places as well, I think.